packages/grid_client/scripts/applications/algorand.ts

```diff
@@ -16,13 +16,6 @@ async function getDeployment(client, vms) {
   log("================= Getting deployment information =================");
 }
 
-async function cancel(client, vms) {
-  const resultVM = await client.machines.delete(vms);
-  log("================= Canceling the deployment =================");
-  log(resultVM);
-  log("================= Canceling the deployment =================");
-}
```

Comment:
why do we remove this method?

Reply:
Unused

Reply:
the usage is commented to not delete the instance once got deployed